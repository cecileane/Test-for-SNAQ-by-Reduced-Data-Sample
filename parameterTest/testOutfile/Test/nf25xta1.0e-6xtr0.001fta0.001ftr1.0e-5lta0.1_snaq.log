optimization of topology, BL and inheritance probabilities using:
 hmax = 1,
 tolerance parameters: ftolRel=1.0e-5, ftolAbs=0.001,
                       xtolAbs=1.0e-6, xtolRel=0.001.
 max number of failed proposals = 25, liktolAbs = 0.1.
rootname for files: nf25xta1.0e-6xtr0.001fta0.001ftr1.0e-5lta0.1_snaq
BEGIN: 2 runs on starting tree (2,3,(4,(5,(1,6):0.386):1.19):0.028);

main seed 52533
seed: 52533 for run 1
Sat Oct  1 17:05:59 2016

 BEGIN SNaQ for run 1, seed 52533 and hmax 1 changed starting topology by NNI move

Begins heuristic optimization of network------

found best network, now we re-optimize branch lengths and gamma more precisely
STOPPED by number of failures criteria
END optTopLevel: found minimizer topology at step 58 (failures: 25) with -loglik=216.1273 and ht_min=[0.39847,0.11024,0.0,0.99131,9.95919]
PERFORMANCE: total number of moves (proposed, successful, accepted) in general, and to fix gamma=0.0,t=0.0 cases
	--------moves general--------				 --------moves gamma,t------	
move	 Num.Proposed	 Num.Successful	 Num.Accepted	 | Num.Proposed	 Num.Successful	 Num.Accepted
add		 2		 2		 1	 |	 NA 		 NA 		 NA 
mvorigin	 16		 16		 1	 |	 0		 0		 --
mvtarget	 15		 15		 1	 |	 0		 0		 --
chdir		 11		 11		 0	 |	 0		 0		 --
delete		 0		 0		 0	 |	 0		 0		 --
nni		 14		 7		 0	 |	 2		 2		 --
Total		 58		 51		 3	 |	 2		 2		 2
Proportion	 -- 		 1.0		 0.1	 |	 -- 		 1.0		 1.0

 FINISHED SNaQ for run 1, -loglik of best 216.12730015767687
((4,2):0.0,(3)#H7:::0.6015285106021506,(1,(6,(5,#H7:::0.3984714893978493):9.959188119841933):0.9913062619829583):0.11023554616414045);
---------------------
seed: 69585 for run 2
Sat Oct  1 17:06:03 2016

 BEGIN SNaQ for run 2, seed 69585 and hmax 1 changed starting topology by NNI move

Begins heuristic optimization of network------

found best network, now we re-optimize branch lengths and gamma more precisely
STOPPED by number of failures criteria
END optTopLevel: found minimizer topology at step 65 (failures: 25) with -loglik=127.5115 and ht_min=[0.45392,0.51861,10.0,0.36538,2.23709,0.0,0.0]
PERFORMANCE: total number of moves (proposed, successful, accepted) in general, and to fix gamma=0.0,t=0.0 cases
	--------moves general--------				 --------moves gamma,t------	
move	 Num.Proposed	 Num.Successful	 Num.Accepted	 | Num.Proposed	 Num.Successful	 Num.Accepted
add		 3		 2		 1	 |	 NA 		 NA 		 NA 
mvorigin	 15		 15		 1	 |	 0		 0		 --
mvtarget	 16		 16		 3	 |	 0		 0		 --
chdir		 14		 14		 1	 |	 0		 0		 --
delete		 0		 0		 0	 |	 0		 0		 --
nni		 17		 9		 0	 |	 2		 2		 --
Total		 65		 56		 6	 |	 2		 2		 2
Proportion	 -- 		 1.0		 0.1	 |	 -- 		 1.0		 1.0

 FINISHED SNaQ for run 2, -loglik of best 127.51150024647512
(2,(1,(6,(5,#H7:0.36538055825154386::0.45391620001107175):10.0):2.2370872408957565):0.5186068630107272,((4,3):0.0)#H7:0.0::0.5460837999889283);
---------------------

Sat Oct  1 17:06:08 2016
MaxNet is (2,(1,(6,(5,#H7:0.36538055825154386::0.45391620001107175):10.0):2.2370872408957565):0.5186068630107272,((4,3):0.0)#H7:0.0::0.5460837999889283); 
with -loglik 127.51150024647512
