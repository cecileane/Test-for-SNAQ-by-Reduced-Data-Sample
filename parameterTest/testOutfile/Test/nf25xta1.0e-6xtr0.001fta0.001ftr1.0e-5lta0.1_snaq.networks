(2,(1,(6,(5,#H7:0.36538055825154386::0.45391620001107175):10.0):2.2370872408957565):0.5186068630107272,((4,3):0.0)#H7:0.0::0.5460837999889283);, with -loglik 127.51150024647512 (best network found, remaining sorted by log-pseudolik; the smaller, the better)
(2,(1,#H7:::0.33015031366617087):1.9490793500076757,(((6)#H7:::0.6698496863338291,5):2.413347433082267,(4,3):0.0):0.8438029183980591);, with -loglik 177.5324503492906
(2,(1)#H7:::0.7256720764763501,(((#H7:::0.2743279235236499,6):5.815308244541829,5):0.7574802148595561,(4,3):0.0):1.5706396121413642);, with -loglik 179.56500300438955
(4,3,(((((2)#H7:::0.5006908817684106,1):3.9533837924623327,6):0.1233809352607994,5):0.20305181540039985,#H7:::0.49930911823158947):0.0);, with -loglik 285.0872138147723
(2,(1,(6,#H7:::0.45880437923424644):2.9831128220340117):0.010760273131234897,((5)#H7:::0.5411956207657536,(4,3):0.0):0.34519908416887946);, with -loglik 303.0082550435201
