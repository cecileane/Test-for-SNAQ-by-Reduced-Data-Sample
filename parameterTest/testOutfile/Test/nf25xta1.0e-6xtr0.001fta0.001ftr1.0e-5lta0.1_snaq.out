(2,(1,(6,(5,#H7:0.36538055825154386::0.45391620001107175):10.0):2.2370872408957565):0.5186068630107272,((4,3):0.0)#H7:0.0::0.5460837999889283); -Ploglik = 127.51150024647512
 Dendroscope: (2,(1,(6,(5,#H7:0.36538055825154386):10.0):2.2370872408957565):0.5186068630107272,((4,3):0.0)#H7:0.0);
 Elapsed time: 9.659535881 seconds in 2 successful runs
-------
List of estimated networks for all runs (sorted by log-pseudolik; the smaller, the better):
 (2,(1,(6,(5,#H7:0.36538055825154386::0.45391620001107175):10.0):2.2370872408957565):0.5186068630107272,((4,3):0.0)#H7:0.0::0.5460837999889283);, with -loglik 127.51150024647512
 ((4,2):0.0,(3)#H7:::0.6015285106021506,(1,(6,(5,#H7:::0.3984714893978493):9.959188119841933):0.9913062619829583):0.11023554616414045);, with -loglik 216.12730015767687
-------
